pkg/lib/bundle/generate.go

```diff
@@ -33,6 +33,11 @@ type AnnotationMetadata struct {
 	Annotations map[string]string `yaml:"annotations"`
 }
 
+type ClusterServiceVersion struct {
+	// Type metadata.
+	Kind string `json:"Kind"`
```

Comment:
Kube resources use lowercase kind metadata keys right?

```suggestion
	Kind string `json:"kind"`
```